scripts/apps/authoring/authoring/index.ts

```diff
@@ -359,20 +360,35 @@ angular.module('superdesk.apps.authoring', [
                 priority: 50,
                 icon: 'kill',
                 group: 'corrections',
-                controller: ['data', 'authoring', 'familyService',
-                    (data, authoring, familyService) => {
+                controller: ['data', 'authoring', 'familyService', 'notify',
+                    (data, authoring, familyService, notify) => {
                         const item = data.item;
                         let relatedItems = [];
 
+                        const handleError = (reason) => {
+                            if (reason != null && reason.status === 400 && get(reason, 'data._issues')) {
+                                notify.error(gettext(`This item is in a package.
```

Comment:
Use a message received from the server for displaying a notification. Just because it's error 400 and has some issues in the payload doesn't mean it's a package related issue. It might be the case at this point in time, but it's naive to think we won't change publishing logic in the future.

Reply:
you know what - you might be right :wink: